fastlane/Helpers/Version.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Version.swift
+//  FastlaneRunner
+//
+//  Created by Khanh on 26/09/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+enum Version {
+    
+    // MARK: - Getting
+
+    static func getVersionNumber() -> String {
+        FastlaneRunner.getVersionNumber(
+            xcodeproj: .userDefined(Constant.projectPath),
+            target: .userDefined(Constant.projectName)
+        )
+    }
+
+    static func getBuildNumber() -> String {
+        FastlaneRunner.getBuildNumber(
+            xcodeproj: .userDefined(Constant.projectPath)
+        )
+    }
+
+    static func getVersionAndBuildNumber() -> String {
+        "\(getVersionNumber()) (Build: \(getBuildNumber())"
+    }
+
+    static var releaseTag: String {
+        "release/\(getVersionNumber())/\(numberOfCommits())"
```

Comment:
For tag, normally, we don't use `numberOfCommits()`. Let's use only version number
```suggestion
        "release/\(versionNumber)"
```